vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -91,7 +91,7 @@ namespace Microsoft::React {
 WinRTWebSocketResource::WinRTWebSocketResource(
     IMessageWebSocket &&socket,
     Uri &&uri,
-    vector<ChainValidationResult> certExeptions) noexcept
+    vector<ChainValidationResult> &&certExeptions) noexcept
```

Comment:
typo

```suggestion
    vector<ChainValidationResult> &&certExceptions) noexcept
```

(same in the other ones)

Reply:
you'll need to fix the one in line 99 (and please fix 114-115 too), the github suggestion thing doesn't let me add a change for an unchanged line it seems